src/OpenTelemetry.Exporter.Prometheus/PrometheusExporter.cs

```diff
@@ -36,18 +36,53 @@ public class PrometheusExporter : MetricExporter
         public PrometheusExporter(PrometheusExporterOptions options)
         {
             this.Options = options;
-            this.Metrics = new List<Metric>();
+            this.LongMetrics = new List<Metric<long>>();
+            this.DoubleMetrics = new List<Metric<double>>();
         }
 
-        internal List<Metric> Metrics { get; private set; }
+        internal List<Metric<long>> LongMetrics { get; private set; }
```

Comment:
I forgot why it's an internal variable and not a temporary variable used during the request from Prometheus processing. Do you have an answer?

Reply:
I realized it can be private. There is a GetAndClear method which returns this list to the PrometheusHttpserver.

I'll revisit this when we implement the final spec for Controller (https://github.com/open-telemetry/opentelemetry-dotnet/issues/390)
Current implementation has Batcher pushing metric (i.e caling Export()) at fixed time intervals. And PrometheusExporter stores the exported metric list in this temp list, which is read and cleared in response to a prometheus scrape call.